### PR TITLE
[Zaraz] Ecommerce - Add Shipping Info Entered and payment_type

### PR DESCRIPTION
**Pull request by omarmosid**
Adds a new 'Shipping Info Entered' event and 'payment_type' API parameter in the Zaraz Ecommerce docs

### CODE REVIEW

**Comment by omarmosid** [comment]
cc: @jonnyparris

File: content/zaraz/web-api/ecommerce.md [hunk 1]
```diff
@@ -106,6 +107,7 @@ Parameter | Type | Description |
 `creative`               | String | Label for creative asset of promotion being tracked.
 `query`                  | String | Product search term.
 `step`                   | Number | The Number of the checkout step in the checkout process.
+`payment_type`           | String | The type of payment used. Maps to `payment_method` in GA4.
```
Comment (jonnyparris) [inline content/zaraz/web-api/ecommerce.md]: Let's keep our docs generic without mentioning any specific tools.
```suggestion
`payment_type`           | String | The type of payment used.
```

**Review by jonnyparris — APPROVED** [review]

**Comment by jonnyparris** [comment]
@haleycode Please can you help getting this merged and approved?

**Review by pedrosousa — APPROVED** [review]

**Comment by jonnyparris** [comment]
thanks @pedrosousa !

**Pull request merged**